src/Tizen.WebView/Tizen.WebView/Context.cs

```diff
@@ -94,5 +106,38 @@ public CookieManager GetCookieManager()
             }
             return _cookieManager;
         }
+
+        /// <summary>
+        /// Clears HTTP caches in the local storage and all resources cached in memory.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void ClearResourceCache()
+        {
+            Interop.ChromiumEwk.ewk_context_resource_cache_clear(_handle);
+        }
+
+        /// <summary>
+        /// Notify low memory to free unused memory.
```

Comment:
Need more detail description
Who is receive this notify? web engine?
Is it request of GC to web engine?

Reply:
.net app also need this API. 
But need more detail description

Reply:
Informs the webengine of a low memory to release a unused memory

Reply:
OK, we will update summary.

Reply:
Done